app/src/main/java/camp/nextstep/edu/tictactoe/ImgBindingAdapter.kt

```diff
@@ -0,0 +1,23 @@
+package camp.nextstep.edu.tictactoe
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.example.tictectoe_domain.Cell
+
+object ImgBindingAdapter {
+    @BindingAdapter("imgRes")
+    @JvmStatic
+    fun loadImage(imageView: ImageView, cell: Cell) {
+        when (cell) {
+            Cell.PLAYER1 -> {
+                imageView.setImageResource(R.drawable.ic_x_black)
+            }
+            Cell.PLAYER2 -> {
+                imageView.setImageResource(R.drawable.ic_o_black)
+            }
+            Cell.NONE -> {
+                imageView.setImageResource(0)
+            }
+        }
+    }
+}
```

Comment:
```suggestion
@BindingAdapter("imgRes")
fun loadImage(imageView: ImageView, cell: Cell) {
    when (cell) {
        Cell.PLAYER1 -> {
            imageView.setImageResource(R.drawable.ic_x_black)
        }
        Cell.PLAYER2 -> {
            imageView.setImageResource(R.drawable.ic_o_black)
        }
        Cell.NONE -> {
            imageView.setImageResource(0)
        }
    }
}
```

Nit: 큰 의미가 없을 땐 object로 감싸주지 않고 최상위 함수로 두어도 괜찮습니다.